test/api/iterations_test.rb

```diff
@@ -132,4 +89,56 @@ def test_skip_problem_as_guest
     assert_equal 401, last_response.status
     assert last_response.body.include?(expected_message)
   end
+
+  private
+
+  def create_submission!(user: nil)
+    Submission.create!(
+      user: user,
+      language: 'ruby',
+      slug: 'one',
+      code: 'CODE1RB',
+      state: 'pending',
+      filename: 'one.rb'
+    )
+  end
+
+  def create_submissions!(user: nil)
```

Comment:
Okay, I get your point. I'll move the data back into the tests. Just a side-note: Maybe it wouldn't be the worst idea to add things like this to the contribution guidelines?